code/modules/mob/living/simple_animal/constructs.dm

```diff
@@ -221,7 +222,7 @@
 	melee_damage_lower = 25
 	melee_damage_upper = 25
 	attacktext = "slashes"
-	speed = -1
+	speed = 0

```

Comment:
Shouldn't need to be redefined since it's already 0 for /mob/living/simple_animal/construct.